Source/astcenccli_entry.cpp

```diff
@@ -254,6 +255,36 @@ static bool validate_cpu_isa()
 	return true;
 }
 
+// Validate Arm SVE availability
+#elif ASTCENC_SVE != 0
+
+#include <sys/auxv.h>
+static bool cpu_supports_sve_256()
+{
+    long hwcaps = getauxval(AT_HWCAP);
```

Comment:
Presumably checking the h/w vector length is not required here?

Reply:
Actually it looks like you are checking the width later, so perhaps change the name?